Source/loadsave.cpp

```diff
@@ -685,6 +685,7 @@ void LoadMissile(LoadHelper *file, Missile &missile)
 	missile.var6 = file->NextLE<int32_t>();
 	missile.var7 = file->NextLE<int32_t>();
 	missile.limitReached = file->NextBool32();
+	missile.lastCollisionTargetHash = 0;

```

Comment:
There are 3 bytes available from alignment, afaik they will always be zero for vanilla saves and it would allow us to retain the sate past saves.

Reply:
I'm not sure if we should use it for this atm, cause
- DevilutionX (includes Version 1.2.1) writes uninitialized data to this (was changed in #2287). So I think it would be good to have version 1.3 out so that most saves should contain zero initialized values (saved in vanilla or devilutionx 1.3).
- The effect is not that great. At maximum you get hit twice once.
- I feel like this bytes are precious and perhaps we could use them for something even better. :innocent: